test/communication/EventFeed.AspNet.Tests.Integration/APageRequest.cs

```diff
@@ -41,10 +43,10 @@ public async Task With_bad_request_returns_links()
             using var host = await A.Host.Build();
             var nonExistentPage = 2;
             var response = await host.GetTestClient().GetAsync($"/api/event-feed/page/{nonExistentPage}");
-            var badRequest = JsonSerializer.Deserialize<BadRequestContent>(await response.Content.ReadAsStringAsync());
-            Assert.Equal("/api/event-feed", badRequest._links._meta);
-            Assert.Equal("/api/event-feed/page/1", badRequest._links._head);
-            Assert.Equal("/api/event-feed/page/1", badRequest._links._tail);
+            var badRequest = JsonSerializer.Deserialize<BadRequestContent>(await response.Content.ReadAsStringAsync(), _jsonSerializerOptions);
+            Assert.Equal("/api/event-feed", badRequest.Links.Meta);
+            Assert.Equal("/api/event-feed/page/1", badRequest.Links.Head);
+            Assert.Equal("/api/event-feed/page/1", badRequest.Links.Tail);
```

Comment:
Another cost, in my opinion, is the tests no longer show as explicitly the expectation. They show the expected value but not the expected JSON structure.

Reply:
Might be a good case for some contract tests.